tests/Resources/Page/OCPDashboard/Builds/Builds.robot

```diff
@@ -43,7 +44,10 @@ Check Image Build Status
 Search Last Build
     [Documentation]    Returns latest(sorted by creation time) build which match the ${build_name_includes}
     [Arguments]    ${namespace}    ${build_name_includes}
-    ${build} =    Run    oc get builds --sort-by=.metadata.creationTimestamp -n ${namespace} | grep ${build_name_includes} | awk '{print $1;}'
+    ${build} =    Run    oc get builds --sort-by=.metadata.creationTimestamp -n ${namespace} | grep ${build_name_includes} | tail -n 1 | awk '{print $1;}'
```

Comment:
this can be done with OpenshiftLibrary so you can avoid error type 2

Reply:
This keyword already existed and I just fixed a bug when more than one build was found. I prefer not to change it now

Reply:
To do the same with OpenshiftLibrary you just need to look into the field status.completionTimestamp in the build

Reply:
As discussed in Google Chat, I'll make this change in another PR